src/main/java/io/zold/api/Copies.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package io.zold.api;
+
+import io.zold.api.Copies.Copy;
+import org.cactoos.iterable.IterableEnvelope;
+import org.cactoos.iterable.IterableOf;
+import org.cactoos.iterable.Mapped;
+
+/**
+ * Multiple copies of a Wallet.
+ *
+ * @since 1.0
+ * @todo #41:30min The constructor of Copies should be implemented
```

Comment:
@victornoel I disagree here a little, we should have a couple of constructors: one with the default behavior where we retrieve the wallet per ID in the remotes as you are proposing here, and another where we can directly supply the wallets for unit testing purposes. Makes sense?

Reply:
@victornoel ping

Reply:
@carlosmiranda the problem I see is that in `Copies`, one `Copy` (a copy being defined as "having the same content") of the wallet is associated to many nodes, and its score is based on those many nodes.

If I follow your advice, 1) I still have to store the score with the copy and 2) it's going a data bag more than an object that relies on other objects.

I was expecting that for testing, `Copies` being just an `Iterable` of `Copy`, you would just build the desired `Iterable` by creating `Copy`s as needed.

Reply:
@victornoel right, that makes sense.